rdf-star-cg-spec.html

```diff
@@ -861,6 +861,24 @@ <h2>Interpolation lemma</h2>
 
   </section>
 
+  <section class="appendix">
+    <h2>Historical remarks</h2>
+
+    <section class="appendix">
+      <h2>SA-mode and PG-mode</h2>
+
+      <p>A lot of discussions on the <a href="https://lists.w3.org/Archives/Public/public-rdf-star/">RDF* mailing list</a> and <a href="https://github.com/w3c/rdf-star">GitHub repository</a> refer to SA-mode and PG-mode. Those abbreviations stand for "Separate Assertion mode" and "Property Graph mode". They originate in the fact that different versions of RDF* have been published over the years, with different designs. In PG-mode, any <a>embedded triple</a> was also considered <a>asserted</a>. SA-mode, on the other hand, allowed to mention <a>embedded triples</a> without asserting them, requiring them to be asserted separately, if that is the intent. SA-mode was more flexible, but induced redundancy in the use-cases that PG-mode was designed to address.</p>
+
+      <p>The notion of <a href="#grammar-production-annotation">annotations</a> in the <a href="#turtle-star">Turtle*</a> syntax was introduced to remove the need for different modes. Instead of interpreting the same syntax differently (which would have caused interoperability problems), it was decided to provide two different syntaxes for each use case.</p>
```

Comment:
```suggestion
      <p>The notion of <a href="#grammar-production-annotation">annotations</a> in the <a href="#turtle-star">Turtle*</a> syntax was introduced to remove the need for different modes. Rather than interpret the same syntax differently in each mode, which would have caused interoperability problems and required a switch for those modes, it was decided to provide a different syntax for each use case.</p>
```